test/lib/deployment/test-deployment.js

```diff
@@ -52,7 +53,11 @@ async function testDeployment(
     );
   }
 
-  const nowJson = JSON.parse(bodies['now.json']);
+  const nowJson = JSON.parse(
+    // we use json5 to allow comments for probes and re-stringify
+    // to make sure it's still valid JSON other than the comments
+    JSON.stringify(json5.parse(bodies['now.json']))
```

Comment:
Why are we stringifying then parsing again

Reply:
> // we use json5 to allow comments for probes and re-stringify
    // to make sure it's still valid JSON other than the comments

Reply:
Won't `JSON.stringify` create valid JSON relardlessless because it's reading an object

Reply:
Ah yeah the extra stringify/parse probably wouldn't catch anything extra, good point